app/screens/home/DetailsScreen.tsx

```diff
@@ -106,6 +107,26 @@ const DetailsScreen = ({route, navigation}: any): any => {
         });
       }
     }
+    else if (fromScreen == "Home") {
+      if(fromCd == true){
+        navigation.navigate('Home', {
+          screen: "Articles",
+          params: {
+            categoryArray: listCategoryArray,
+            backClicked: 'no'
+          },
+          merge: true
+        })
+      }
+      else{
+        // navigation.navigate({
+        //   name: fromScreen == "ChildgrowthTab2" ? "ChildgrowthTab" : fromScreen,
+        //   params: {categoryArray:listCategoryArray,backClicked:'yes'},
+        //   merge: true,
+        // });
```

Comment:
Let's please remove the comment if not neccessary.

Reply:
ok